recipes/gperf/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+
+class GperfConan(ConanFile):
+    name = "gperf"
+    license = "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/gperf"
+    description = "GNU gperf is a perfect hash function generator"
+    topics = ("conan", "gperf", "hash-generator", "hash")
+    settings = "os_build", "arch_build", "compiler"
+    _source_subfolder = "source_subfolder"
+    _autotools = None
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == "Visual Studio"
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os_build == "Windows" and self.settings.compiler == "gcc" and os.name == "nt"
```

Comment:
Is `os.name` required?

Reply:
I think it is required, but we should probably use `tools.os_info.is_windows`

Reply:
mingw is also available on Linux